<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>firstInitialisation</class>
 <widget class="QDialog" name="firstInitialisation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>235</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(42,42,41);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item alignment="Qt::AlignHCenter|Qt::AlignTop">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
font-size:20px;</string>
     </property>
     <property name="text">
      <string>WELCOME!</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_5">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:white;
font-size:10px;</string>
       </property>
       <property name="text">
        <string>Input your username</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit">
       <property name="styleSheet">
        <string notr="true">color:white;
font-size:10px;
border-style: solid;
border-width: 0 0 2px 0; 
border-color: rgb(20, 100, 246);</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="styleSheet">
      <string notr="true">QPushButton{
  background-color: #000000;
  border: 2px solid #1A1A1A;
  border-radius: 15px;
  color: #FFFFFF;
  font-size: 14px;
  font-weight: 600;
  margin: 0;
  min-height: 16px;
  padding: 16px 24px;
  text-align: center;
  transition: all 300ms cubic-bezier(.23, 1, 0.32, 1);
  width: 100%;
}

QPushButton:hover {
  background-color: #333333;
  border: 2px solid #555555;
  box-shadow: 0 8px 15px rgba(0, 0, 0, 0.25);
}

QPushButto:pressed {
  background-color: #FF6347;
         border: 2px solid #FF4500;
        box-shadow: 0 4px 8px rgba(0, 0, 0, 0.5);
}

QPushButton {
  outline: none; 
}</string>
     </property>
     <property name="text">
      <string>START!</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
